lib/lynis/lynistest.pm

```diff
@@ -0,0 +1,289 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Base module for Lynis test cases
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#78224
+
+package lynis::lynistest;
+
+use strict;
+use warnings;
+use testapi;
+use utils;
+use autotest;
+
+use base 'consoletest';
+use LTP::TestInfo 'testinfo';
```

Comment:
Just don't understand her about using LTP here ? could you explain it? thanks.

Reply:
> Just don't understand it about using LTP here ? could you explain it? thanks.

I invoke the object which LTP have defineded so I do not to write duplicated code here. 
see here also: https://github.com/os-autoinst/os-autoinst/blob/master/autotest.pm

Reply:
OKay I got it, thanks.